app/models/manageiq/providers/openstack/manager_mixin.rb

```diff
@@ -46,7 +46,7 @@ def ems_connect?(password, params, service)
       ems.security_protocol      = params[:security_protocol].strip
       ems.keystone_v3_domain_id  = params[:uid_ems]
 
-      user, hostname, port = params[:userid], params[:hostname].strip, params[:port].try(:strip)
+      user, hostname, port = params[:userid], params[:hostname].strip, params[:port]
```

Comment:
This is only called by the `verify_credentials?` method which takes DDF params, if a user does pass in a string port for some reason this still works

Reply:
Out of curiosity, if the port is a string and has, say, leading spaces, will this still work?

Reply:
No, but you also can't provider a port with a leading space in the ddf form